src/migration/1692265909018-VoteTriggers.ts

```diff
@@ -0,0 +1,104 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class VoteTriggers1692265909018 implements MigrationInterface {
+    name = 'VoteTriggers1692265909018'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+      await queryRunner.query('DROP TRIGGER IF EXISTS upvote_create_trigger ON upvote')
+      await queryRunner.query('DROP TRIGGER IF EXISTS upvote_delete_trigger ON upvote')
+
+      await queryRunner.query('DROP TRIGGER IF EXISTS downvote_create_trigger ON downvote')
+      await queryRunner.query('DROP TRIGGER IF EXISTS downvote_delete_trigger ON downvote')
```

Comment:
We remove old triggers to not have `post` be double updated. We don't remove procedures for now to be able to revert migration if needed.